devices/ITCommander.js

```diff
@@ -0,0 +1,24 @@
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const exposes = require('../lib/exposes');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['ITCMDR_Contact'],
+        model: 'ITCMDR_Contact',
+        vendor: 'IT Commander',
+        description: 'Contact Sensor',
+        fromZigbee: [fz.contact],
```

Comment:
`fz.contact` does not exist, what messages does it send for contact?

Reply:
I am sorry give you trubles, I am newbie, I think I fixed, please run workflow again.